### PR TITLE
[DT-038449] OpenAI Module: Workflow Catalog - 'Chat with Chat GPT' Fl…

### PR DESCRIPTION
**Pull request by deci-shawns**
Validation icon is getting cut on "Send" button in Chat Completion form. Updated DecObj to fix.

### CODE REVIEW

**Comment by deci-shawns** [comment]
Merged [DT-038454] OpenAI Moderation Categories Must Be Added to Match Updated Version and updated SDK to 8.13.0

File: Decisions.OpenAI/DataTypes/OpenAiModeration/ModerationCategories.cs
Comment (jason-decisions) [inline Decisions.OpenAI/DataTypes/OpenAiModeration/ModerationCategories.cs:1]: I think we could do a better job at these data types so we're not duplicating properties. You could have a `Moderation` type that has all the common properties between `ModerationCategories` and `ModerationCategoryScores`. Then each of those could extend the `Moderation` class and eliminate the duplicate code.

Comment (jason-decisions) [reply Decisions.OpenAI/DataTypes/OpenAiModeration/ModerationCategories.cs:1]: This should be applied to all Data Types.

Comment (jason-decisions) [reply Decisions.OpenAI/DataTypes/OpenAiModeration/ModerationCategories.cs:1]: Disregard as the types are different.

**Pull request merged**